app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -281,10 +281,11 @@ struct DeviceDelegateWaveVR::State {
       float immersiveAxes[kNumAxes] = { 0.0f, 0.0f };
 
       if (touchpadTouched) {
+        const float speed = 0.7f;
         WVR_Axis_t axis = WVR_GetInputAnalogAxis(controller.type, WVR_InputId_Alias1_Touchpad);
-        delegate->SetTouchPosition(controller.index, axis.x, -axis.y);
-        immersiveAxes[0] = axis.x;
-        immersiveAxes[1] = -axis.y;
+        delegate->SetTouchPosition(controller.index, axis.x * speed, -axis.y * speed);
+        immersiveAxes[0] = axis.x * speed;
+        immersiveAxes[1] = -axis.y * speed;
```

Comment:
We should not be adjusting the scale of the immersiveAxes. The raw data should be passed to the page in immersive mode. Also since there is no time component, the `speed` variable is really a just a scale.